### ISSUE TITLE
getting the following error when starting Monitor.dll

### ISSUE THREAD

**Issue by i1wan** [Legedric/ptmagic#13]
```
root@profi:/var/opt/PTMagic/Monitor# dotnet Monitor.dll
Error:
  An assembly specified in the application dependencies manifest (Monitor.deps.json) was not found:
    package: 'Microsoft.AspNetCore.Antiforgery', version: '2.0.1'
    path: 'lib/netstandard2.0/Microsoft.AspNetCore.Antiforgery.dll'
  This assembly was expected to be in the local runtime store as the application was published using the following target manifest files:
    aspnetcore-store-2.0.3.xml
root@profi:/var/opt/PTMagic/Monitor# 
```

**Comment by innerpartycolt** [comment]
Have you tried to call it from PTMagic folder? like dotnet Monitor/Monitor.dll

**Comment by i1wan** [comment]
Same result:

root@profi:/var/opt/PTMagic# dotnet Monitor/Monitor.dll
Error:
  An assembly specified in the application dependencies manifest (Monitor.deps.json) was not found:
    package: 'Microsoft.AspNetCore.Antiforgery', version: '2.0.1'
    path: 'lib/netstandard2.0/Microsoft.AspNetCore.Antiforgery.dll'
  This assembly was expected to be in the local runtime store as the application was published using the following target manifest files:
    aspnetcore-store-2.0.3.xml
root@profi:/var/opt/PTMagic#

**Comment by innerpartycolt** [comment]
This is strange. Worked for me that way, although the error was a little different. Can you confirm that the file isn't there?

**Comment by i1wan** [comment]
The file is there ... 

root@profi:/var/opt/PTMagic/Monitor# ls -l | grep deps
-rwxrwxrwx 1 administrator administrator  300125 Feb 17 14:40 Monitor.deps.json
root@profi:/var/opt/PTMagic/Monitor#

**Comment by TwixRaider** [comment]
Can you please navigate to the 'Monitor' folder and try the command 'dotnet restore'? 

This should download all dependencies defined in the 'Monitor.deps.json'. If this is done please try to start the monitor application again.

**Comment by TwixRaider** [comment]
Found this link saying that the "Microsoft .NET Core 2.0.0 Linux Server Hosting" needs to be installed on a linux machine. Maybe this helps...

https://serverfault.com/questions/884332/what-package-should-i-install-for-running-a-deployed-net-core-application

**Comment by i1wan** [comment]
"Microsoft .NET Core 2.0.0 Linux Server Hosting" is installed ... see below ... 

root@profi:/var/opt/PTMagic# apt-cache search dotnet
libgtk2.0-cil - CLI binding for the GTK+ toolkit 2.12
libgtk2.0-cil-dev - CLI binding for the GTK+ toolkit 2.12
libgtk-dotnet3.0-cil - GTK.NET library
libgtk-dotnet3.0-cil-dev - GTK.NET library - development files
dotnet-dev-1.0.4 - .NET Core SDK 1.0.4
dotnet-dev-1.1.4 - .NET Core SDK 1.1.4
dotnet-dev-1.1.5 - .NET Core SDK 1.1.5
dotnet-dev-1.1.6 - .NET Core SDK 1.1.6
dotnet-dev-1.1.7 - .NET Core SDK 1.1.7
dotnet-host - Microsoft .NET Core Host - 2.0.5
dotnet-hostfxr-1.0.1 - Microsoft .NET Core 1.0.6 - Host FX Resolver 1.0.1
dotnet-hostfxr-1.1.0 - Microsoft .NET Core 1.1.3 - Host FX Resolver 1.1.0
dotnet-hostfxr-2.0.0 - Microsoft .NET Core Host FX Resolver - 2.0.0 2.0.0
dotnet-hostfxr-2.0.0-preview2-25407-01 - Microsoft .NET Core Host FX Resolver - 2.0.0 Preview 2 2.0.0-preview2-25407-01
dotnet-hostfxr-2.0.3 - Microsoft .NET Core Host FX Resolver - 2.0.3 2.0.3
dotnet-hostfxr-2.0.4 - Microsoft .NET Core Host FX Resolver - 2.0.4 2.0.4
dotnet-hostfxr-2.0.5 - Microsoft .NET Core Host FX Resolver - 2.0.5 2.0.5
dotnet-hosting-2.0.0 - Microsoft .NET Core 2.0.0 Linux Server Hosting
dotnet-hosting-2.0.3 - Microsoft .NET Core 2.0.3 Linux Server Hosting
dotnet-hosting-2.0.4 - Microsoft .NET Core 2.0.4 Linux Server Hosting
dotnet-hosting-2.0.5 - Microsoft .NET Core 2.0.5 Linux Server Hosting
dotnet-runtime-2.0.0 - Microsoft .NET Core Runtime - 2.0.0 Microsoft.NETCore.App 2.0.0
dotnet-runtime-2.0.0-preview2-25407-01 - Microsoft .NET Core Runtime - 2.0.0 Preview 2 Microsoft.NETCore.App 2.0.0-preview2-25407-01
dotnet-runtime-2.0.3 - Microsoft .NET Core Runtime - 2.0.3 Microsoft.NETCore.App 2.0.3
dotnet-runtime-2.0.4 - Microsoft .NET Core Runtime - 2.0.4 Microsoft.NETCore.App 2.0.4
dotnet-runtime-2.0.5 - Microsoft .NET Core Runtime - 2.0.5 Microsoft.NETCore.App 2.0.5
dotnet-sdk-2.0.0 - Microsoft .NET Core SDK - 2.0.0
dotnet-sdk-2.0.0-preview2-006497 - Microsoft .NET Core SDK 2.0.0 Preview 2
dotnet-sdk-2.0.2 - Microsoft .NET Core SDK - 2.0.2
dotnet-sdk-2.0.3 - Microsoft .NET Core SDK - 2.0.3
dotnet-sdk-2.1.2 - Microsoft .NET Core SDK - 2.1.2
dotnet-sdk-2.1.3 - Microsoft .NET Core SDK - 2.1.3
dotnet-sdk-2.1.4 - Microsoft .NET Core SDK - 2.1.4
dotnet-sharedframework-microsoft.netcore.app-1.0.5 - Microsoft .NET Core 1.0.5 - Runtime Microsoft.NETCore.App 1.0.5
dotnet-sharedframework-microsoft.netcore.app-1.0.7 - Microsoft .NET Core 1.0.7 - Runtime Microsoft.NETCore.App 1.0.7
dotnet-sharedframework-microsoft.netcore.app-1.0.8 - Microsoft .NET Core 1.0.8 - Runtime Microsoft.NETCore.App 1.0.8
dotnet-sharedframework-microsoft.netcore.app-1.0.9 - Microsoft .NET Core 1.0.9 - Runtime Microsoft.NETCore.App 1.0.9
dotnet-sharedframework-microsoft.netcore.app-1.1.2 - Microsoft .NET Core 1.1.2 - Runtime Microsoft.NETCore.App 1.1.2
dotnet-sharedframework-microsoft.netcore.app-1.1.4 - Microsoft .NET Core 1.1.4 - Runtime Microsoft.NETCore.App 1.1.4
dotnet-sharedframework-microsoft.netcore.app-1.1.5 - Microsoft .NET Core 1.1.5 - Runtime Microsoft.NETCore.App 1.1.5
dotnet-sharedframework-microsoft.netcore.app-1.1.6 - Microsoft .NET Core 1.1.6 - Runtime Microsoft.NETCore.App 1.1.6
root@profi:/var/opt/PTMagic#

**Comment by i1wan** [comment]
The version (that it uses by default) also looks ok 

root@profi:~# dotnet --version
2.0.2
root@profi:~#

**Comment by i1wan** [comment]
The "dotnet restore" did not help either:

```
root@profi:/var/opt/PTMagic/Monitor# dotnet restore

Welcome to .NET Core!
---------------------
Learn more about .NET Core @ https://aka.ms/dotnet-docs. Use dotnet --help to see available commands or go to https://aka.ms/dotnet-cli-docs.

Telemetry
--------------
The .NET Core tools collect usage data in order to improve your experience. The data is anonymous and does not include command-line arguments. The data is collected by Microsoft and shared with the community.
You can opt out of telemetry by setting a DOTNET_CLI_TELEMETRY_OPTOUT environment variable to 1 using your favorite shell.
You can read more about .NET Core tools telemetry @ https://aka.ms/dotnet-cli-telemetry.
MSBUILD : error MSB1003: Specify a project or solution file. The current working directory does not contain a project or solution file.
root@profi:/var/opt/PTMagic/Monitor# dotnet Monitor.dll 
Error:
  An assembly specified in the application dependencies manifest (Monitor.deps.json) was not found:
    package: 'Microsoft.AspNetCore.Antiforgery', version: '2.0.1'
    path: 'lib/netstandard2.0/Microsoft.AspNetCore.Antiforgery.dll'
  This assembly was expected to be in the local runtime store as the application was published using the following target manifest files:
    aspnetcore-store-2.0.3.xml
root@profi:/var/opt/PTMagic/Monitor# 
```

**Comment by Legedric** [comment]
Did you try to remove dotnet (core) from your machine and start fresh? Maybe something went wrong with the initial install.

Other than that, we are no Unix experts and do not run any Unix project, so our support on that platform is kinda limited to be honest.

**Comment by Zamboniman** [comment]
I'm getting the same error.

**Comment by krims0n32** [comment]
Upgrade your dotnet version. I had the same issue. Fixed when using:

# dotnet --version
2.1.4

**Comment by Zamboniman** [comment]
The latest core seems to be 2.0.5, and the latest SDK is 2.1.4. This is what I have and this does not help, I still get this error.

**Comment by eiertee** [comment]
Hi there! On my side the monitor .dll is not there altogether ... 'No executable found matching command "dotnet-Monitor/Monitor.dll". PLease advise

**Comment by Zamboniman** [comment]
There shouldn't be a hyphen between dotnet and Monitor/Monitor.dll

It's as follows:

`dotnet Monitor/Monitor.dll`

Assuming you are in your PT Magic directory.

**Comment by Zamboniman** [comment]
See Closed issue #65, see if this solves this one as well, as they appear to be the same.

**Comment by Legedric** [comment]
Closing this as there was no feedback for a couple of days from the original poster.
